packages/web/tests/pages/app/contributions.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import fetchMock from 'fetch-mock-jest';
+import { render, screen, act } from '../../testUtils/testTools';
+import { getMock } from '../../testUtils/getMock';
+import { AppLayout } from '../../../src/components/Layout';
+import Contributions from '../../../src/pages/app/contributions';
+import { ContributionsBox } from '../../../src/components/Contributions';
+import { ContributionsList } from '../../../src/pages/app/contributions';
+
+jest.mock('../../../src/components/Layout/AppLayout.tsx');
+getMock(AppLayout).mockImplementation(({ children }) => <>{children}</>);
+
+jest.mock('../../../src/components/Contributions/ContributionsBox.tsx');
+getMock(ContributionsBox).mockImplementation(() => <p>Box Row</p>);
+
+const clist1: ContributionsList = {
+  id: '1',
+  nodeID: 'PR_00000',
+  description: 'Pizza but a construct of the mind',
+  type: 'CLOSED',
+  score: 1,
+  contributedAt: new Date('1612-04-20'),
+  url: 'abc.com',
+};
+
+const clist2: ContributionsList = {
+  id: '2',
+  nodeID: 'PR_54321',
+  description: 'Count 5 to 1',
+  type: 'OPEN',
+  score: 1,
+  contributedAt: new Date('2022-09-12'),
+  url: 'onetofive.com',
+};
```

Comment:
Now that you have two contributions, you can make a contribution list:

```suggestion
const contributionList: Contribution[] = [{
  id: '1',
  nodeID: 'PR_00000',
  description: 'Pizza but a construct of the mind',
  type: 'CLOSED',
  score: 1,
  contributedAt: new Date('1612-04-20'),
  url: 'abc.com',
},
{
  id: '2',
  nodeID: 'PR_54321',
  description: 'Count 5 to 1',
  type: 'OPEN',
  score: 1,
  contributedAt: new Date('2022-09-12'),
  url: 'onetofive.com',
}];
```